ftw/simplelayout/aliasblock/profiles/default/types/ftw.simplelayout.AliasBlock.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0"?>
+<object name="ftw.simplelayout.AliasBlock"
+        meta_type="Dexterity FTI"
+        xmlns:i18n="http://xml.zope.org/namespaces/i18n"
+        i18n:domain="ftw.simplelayout" >
+
+    <!-- Basic metadata -->
+    <property name="title" i18n:translate="">AliasBlock</property>
+    <property name="description" i18n:translate="aliasblock_type_description"></property>
+    <property name="icon_expr"></property>
+    <property name="allow_discussion">False</property>
+    <property name="global_allow">False</property>
+    <property name="filter_content_types">True</property>
+    <property name="allowed_content_types" />
+
+    <!-- schema interface -->
+    <property name="schema">ftw.simplelayout.aliasblock.contents.aliasblock.IAliasBlockSchema</property>
+
+    <!-- class used for content items -->
+    <property name="klass">ftw.simplelayout.aliasblock.contents.aliasblock.AliasBlock</property>
+
+    <!-- add permission -->
+    <property name="add_permission">ftw.simplelayout.AddAliasBlock</property>
+
+    <!-- enabled behaviors -->
+    <property name="behaviors">
+        <element value="plone.app.content.interfaces.INameFromTitle" />
+        <element value="ftw.simplelayout.interfaces.ISimplelayoutBlock" />
+    </property>
```

Comment:
Just thought whether this needs to be different in Plone 4......  From looking at bobtemplates.plone it does

In particular
https://github.com/plone/bobtemplates.plone/blob/master/bobtemplates/plone/content_type/profiles/default/types/%2Bdexterity_type_fti_file_name%2B.xml.bob#L70

shows Referenceable behavior (which is also used for e.g. https://github.com/4teamwork/ftw.simplelayout/blob/master/ftw/simplelayout/contenttypes/profiles/default/types/ftw.simplelayout.ContentPage.xml#L39) 

    <element value="plone.app.referenceablebehavior.referenceable.IReferenceable" />

Reply:
The referenceable behavior is used for DX to AT references, this not necessary here, since we only allow DX types as references.

Reply:
Ok - so it should not be in ftw.simplelayout.ContentPage.xml then ? (though I guess it doesn't hurt)